app/sales-api/handlers/handlers.go

```diff
@@ -29,6 +30,11 @@ func API(build string, shutdown chan os.Signal, log *log.Logger, a *auth.Auth, d
 	app.HandleDebug(http.MethodGet, "/readiness", cg.readiness)
 	app.HandleDebug(http.MethodGet, "/liveness", cg.liveness)
 
+	// Accept CORS 'OPTIONS' preflight requests
```

Comment:
Can we comment this code with a good set of notes above. Talk about the need to add the middleware and more of what this does, how and why?